libminifi/src/c2/C2Agent.cpp

```diff
@@ -521,9 +545,9 @@ void C2Agent::handle_describe(const C2ContentResponse &resp) {
     std::vector<std::string> keys;
     std::copy_if(unsanitized_keys.begin(), unsanitized_keys.end(), std::back_inserter(keys), [](std::string key) {return key.find("pass") == std::string::npos;});
     C2Payload response(Operation::ACKNOWLEDGE, resp.ident, false, true);
-    response.setLabel("configuration_options");
+    response.setLabel("configurationOptions");
```

Comment:
Why was this changed? Doesn't this break backward compatibility? (I guess if DESCRIBE has not been formalized yet it doesn't, but I want to make sure.)

Reply:
Since a release was performed it's technically possible someone used this ( semver on an open source Apache project ) -- so we should maintain it; however, it's not defined in https://cwiki.apache.org/confluence/display/MINIFI/C2+Design specifically as configuration_options. Previously the goal was to fill that in when some of this was finalized.

Reply:
Agree. The wiki doc is very much out of date and we cannot tell from that if someone might be using it. Will revert the label name.